microsoft-edge/devtools-guide-chromium/whats-new/2024/04/devtools-124.md

```diff
@@ -0,0 +1,100 @@
+
+
+<!-- ====================================================================== -->
+## postMessage events and handlers are distinguished from other events on the timeline
+
+<!-- Subtitle: Easily investigate performance issues related to posting messages across various threads in applications.  -->
+
+A new experiment improves the **Performance** tool's timeline to help you quickly identify `postMessage` events and handlers, by distinguishing `postMessage` events from other timeline events.  Before, `postMessage` dispatch events and handler events both appeared as generic scripting function call events.  Now:
+* `postMessage` dispatch events appear on the timeline as **Schedule postMessage**.
+* `postMessage` handler events appear on the timeline as **On Message**.
+
+!["Schedule postMessage" events and "On Message" events in the Performance tool](./devtools-124-images/post-message-events.png)
+
+The performance timeline has been updated to help you investigate when a `postMessage` call occurred, and how long the message was queued before the handler starts.  The dispatch events are linked to handler events by initiator arrows that appear when you click on either type of event:
+
+![Arrows linking dispatch events to handler events](./devtools-124-images/post-message-events-arrows.png)
+
+To use this feature, in DevTools, select **Customize and control DevTools** (![the Customize and control DevTools icon](./devtools-124-images/customize-and-control-devtools-icon.png)) > **Settings** > **Experiments**, select the checkbox **Timeline: Show postMessage dispatch and handling flows**, click the **Close** (X) button, and then click the **Reload DevTools** button.
+
+See also:
+* [Timeline event reference](/microsoft-edge/devtools-guide-chromium/evaluate-performance/performance-reference)
+
+
+<!-- ====================================================================== -->
+## Move a tab left or right in the Activity bar by right-clicking it
+
+<!-- Subtitle: Move the tabs in the Activity bar left or right by using the tab's right-click menu. -->
+
+Before Microsoft Edge 124, you could rearrange the tabs in the **Activity bar** only by dragging them.  This can be difficult for users who are using input devices such as a trackball or head pointer.  Now, you can also move a tab left or right by right-clicking the tab and then selecting **Move left** or **Move right**:
+
+![The context menu from right-clicking a tool's tab on the Activity bar](./devtools-124-images/rearrange-tab.png)
+
+See also:
+* [Pin and rearrange tools in the Activity Bar](/microsoft-edge/devtools-guide-chromium/overview#pin-and-rearrange-tools-in-the-activity-bar) in _Overview of DevTools_.
+
+
+<!-- ====================================================================== -->
+## Network tab shows a warning icon when using overrides or throttling
+
+<!-- Subtitle: Previously, this only worked in DevTools' legacy UI, but it's been fixed to work in the new UI now. -->
```

Comment:
in body text, i wrote-around to avoid drawing attention to "legacy UI" vs [focus mode] UI, since we haven't formalized these contrasting terms.  This Subtitle string lacks any discussion of UI specifics, but puts all focus on contrast between (undefined term) legacy UI vs. (unnamed) Focus Mode UI.  Maybe rewrite Subtitle to avoid discussing in terms of old vs new UI.
@captainbrosset @leahmsft

Reply:
result:
```
<!-- Subtitle: The warning icon on the Network tab when overrides or throttling is enabled has been restored. -->
```